### PR TITLE
Add a few more extensions.

### PR DESCRIPTION
**Pull request by ben-eb**
Corrected duplicated `flac` to `fla` (assumed this is what you meant?). Added some more extensions that I could think of off the top of my head. :smile:

Side note; may be worth categorising these? There are a _lot_ of different types...

### CODE REVIEW

File: binary-extensions.json [hunk 1]
```diff
@@ -25,10 +29,12 @@
 	"ecelp9600",
 	"eol",
 	"eot",
+	"epub",
+	"exe",
 	"f4v",
 	"fbs",
 	"fh",
-	"flac",
```
Comment (sindresorhus) [inline binary-extensions.json:31]: no http://en.wikipedia.org/wiki/FLAC

**Comment by sindresorhus** [comment]
Thanks @ben-eb :)

File: binary-extensions.json (hunk 1 above)
Comment (ben-eb) [inline binary-extensions.json:31]: I meant it is mentioned twice (https://github.com/sindresorhus/binary-extensions/blob/master/binary-extensions.json#L31-L32), so I corrected one of them to point to Adobe's `fla` format instead.

Comment (sindresorhus) [inline binary-extensions.json:31]: Oh, sorry, that confused me.

**Pull request merged**